go.sum

```diff
@@ -157,6 +159,8 @@ github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 h1:EzJWgHo
 github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415/go.mod h1:GwrjFmJcFw6At/Gs6z4yjiIwzuJ1/+UwLxMQDVQXShQ=
 github.com/xeipuuv/gojsonschema v1.1.0 h1:ngVtJC9TY/lg0AA/1k48FYhBrhRoFlEmWzsehpNAaZg=
 github.com/xeipuuv/gojsonschema v1.1.0/go.mod h1:5yf86TLmAcydyeJq5YvxkGPE2fm/u4myDekKRoLuqhs=
+github.com/xi2/xz v0.0.0-20171230120015-48954b6210f8 h1:nIPpBwaJSVYIxUFsDv3M8ofmx9yWTog9BfvIu0q41lo=
+github.com/xi2/xz v0.0.0-20171230120015-48954b6210f8/go.mod h1:HUYIGzjTL3rfEspMxjDjgmT5uz5wzYJKVo23qUhYTos=
```

Comment:
It looks like [something](https://community-tc.services.mozilla.com/tasks/E5yVuT0JQeaIInsNnG4-BA/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FE5yVuT0JQeaIInsNnG4-BA%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L2003) is missing here.

Reply:
> It looks like [something](https://community-tc.services.mozilla.com/tasks/E5yVuT0JQeaIInsNnG4-BA/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FE5yVuT0JQeaIInsNnG4-BA%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L2003) is missing here.

I have no idea how to fix that

Reply:
Those appeared when I applied `go get -u github.com/mholt/archiver` to update to latest version ...

Reply:
> I have no idea how to fix that
Running `go mod tidy` should do it.